bin-src/lib/getDependentStoryFiles.ts

```diff
@@ -145,9 +150,8 @@ export async function getDependentStoryFiles(
   function files(moduleName: string) {
     const mod = modulesByName[moduleName];
     if (!mod) return [moduleName];
-    return mod.modules && MULTI_MODULES.test(mod.name)
```

Comment:
I'd like to understand why this regex test was needed.  The naming for the vite modules does not include it.